api/graphql/schema.graphql

```diff
@@ -615,7 +615,8 @@ type Query {
     parentSlugs: [String],
     search: String,
     sortBy: String,
-    visibility: Int
+    visibility: Int,
+    coord: PointInput
```

Comment:
let's alphabetize

Reply:
also if this param always means "near this coordinate" then i wonder if we can come up with a more descriptive name. nearCoord? aroundCoord?

Reply:
Updated